docs/detections/rules-ui-create.asciidoc

```diff
@@ -123,11 +134,11 @@ image::images/rule-start-ml-job.png[]
 ==== Create a custom query rule
 . To create a rule based on a KQL or Lucene query, select *Custom query*,
 then:
-.. Define which {es} indices the rule searches for alerts.
+.. Define which {es} indices or data view the rule searches for alerts.
 .. Use the filter and query fields to create the criteria used for detecting
 alerts.
 +
-NOTE: You can use {kib} saved queries (save icon) and queries from saved Timelines (`Import query from saved Timeline`) as rule conditions.
+NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,17,17]) and queries from saved Timelines (`Import query from saved Timeline`) as rule conditions.
```

Comment:
I think the Timelines link should be bold here, since it's a label/link the user will click. Monospace is typically more for values that the user enters, or code.
```suggestion
NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,17,17]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
```